### DISCUSSION TITLE
Correct import of templates

### DISCUSSION THREAD

**Discussion by savchenko** [helloflask/bootstrap-flask#362] [Q&A]
```py
app = Flask(__name__)
bootstrap = Bootstrap5(app)
```

does not expose template macros to the project. 

Of course it's possible to symlink from the cloned repository, but this is less than optimal for obvious reasons.

Am I missing something obvious here?

**Comment by greyli** [comment]
Did you get any error outputs?

**Reply by savchenko** [reply]
Nothing unusual (apart from the macro not being found). Last stable Flask, example app launches just fine.

**Reply by greyli** [reply]
Then what's the error output for "the macro not being found"?  What's your import statement in your template?
